repos/vstest.proj

```diff
@@ -17,7 +17,6 @@
     <PackagesOutput>$(ProjectDirectory)/artifacts/$(Configuration)/packages</PackagesOutput>
     <RepoApiImplemented>false</RepoApiImplemented>
     <DeterministicBuildOptOut>true</DeterministicBuildOptOut>
-    <GlobalJsonFile>$(ProjectDirectory)global.json</GlobalJsonFile>
```

Comment:
Not sure why I didn't notice this earlier... this seems strange, it should disable some of our override logic. Why is it removed?

Reply:
I'll add it back. I was not sure if I could keep it since the global.json file specified a 3.1 sdk. What gets done to the file specified here?

Reply:
It has a 5.0 SDK now: https://github.com/microsoft/vstest/blob/b195e2589980861425b331e73a859252c3f2b71a/global.json. But even with a 3.1 SDK, we need this--source-build overrides it with the 5.0 SDK that we can actually use at the time. (3.1 would be a prebuilt that the repo would try to download, or if we block the download, the repo would fail to build.) This could be the prebuilt tooling version for prod builds, but also an N-1, or an N version for bootstrapping.